### ISSUE TITLE
why not train a neural network?

### ISSUE THREAD

**Issue by IpekUyguner** [cc-hpc-itwm/DeepFakeDetection#11]
Is there a specific reason for not training NN for the data sets in which the accuracy is lower than 100%,  like DeepFakeDetection dataset?

**Comment by RicardDurall** [comment]
Hello, actually there is no specific reason. We haven't used NN because we wanted to show the power of our features on "simple" classifiers. Probably you could improve slightly the score by using NN but you should be careful with overfitting.
